config/manifests/bases/istio-workspace-operator.clusterserviceversion.yaml

```diff
@@ -21,6 +21,12 @@ spec:
       displayName: Session
       kind: Session
       name: sessions.maistra.io
+      statusDescriptors:
+      - description: Status of resources deployed/modifed by this Session resource
+        displayName: Resources
+        path: readiness.components
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podStatuses
```

Comment:
I wonder at which point in time `com.tectonic.ui` will become `io.openshift.ui` and we won't notice ;)

Reply:
heh yea.. :)